.env.development

```diff
@@ -120,6 +120,6 @@ OWNER_EMAILS='Owner <owner@test.com>, Another Owner <another-owner@test.com>'
 PROGRAM_INCEPTION_DATE=2018-01-01
 SUPERVISOR_EMAILS='Supervisor <supervisor@test.com>'
 
-SEED_GENERATED_ADMIN_PASSWORD='baby monkey honey badger star'
```

Comment:
I'm not sure if there's a reason two of the seed passwords are different than the rest. I've made them all the same for simplicity, but I can make changes if there's a reason to keep them different.

Reply:
Sounds good -- this particular password was a holdover from earlier seed code.

When you do ship this, please make sure to let every dev/designer know that their seed users will have different passwords the next time they do a reseed.

Reply:
👍